client/data/help/use-help-search-query.ts

```diff
@@ -59,22 +47,9 @@ export const useHelpSearchQuery = (
 	sectionName = ''
 ) => {
 	return useQuery< any >( {
-		queryKey: [ 'help-center-search', search, sectionName ],
+		queryKey: [ 'help-center-search', search, locale, sectionName ],
 		queryFn: () => fetchArticlesAPI( search, locale, sectionName ),
-		onSuccess: async ( data ) => {
-			await Promise.all(
-				data.map( async ( result: SearchResult ) => {
```

Comment:
This code was't doing anything and is seems to be a leftover from https://github.com/Automattic/wp-calypso/pull/77955 so we remove it. `onSuccess` handlers for query hooks will be removed in v5 of RQ.